ui/packages/tidb-dashboard-lib/src/apps/Monitoring/components/Monitoring.tsx

```diff
@@ -52,6 +51,19 @@ export default function Monitoring() {
     return setTimeRange((r) => ({ ...r }))
   }
 
+  const handleOnBrush = (range: TimeRangeValue) => {
+    setChartRange(range)
+  }
+
+  const ErrorComponent = (error: Error) => (
+    <Space direction="vertical">
+      <ErrorBar errors={[error]} />
+      <Link to="/user_profile?blink=profile.prometheus">
+        {t('components.metricChart.changePromButton')}
+      </Link>
```

Comment:
in `tidb-dashboard-for-dbaas` and `tidb-dashboard-for-clinic-op` variant, we don't have `/user_profile` route.